relay/bridgeStub.go

```diff
@@ -0,0 +1,111 @@
+package relay
+
+import (
+	"context"
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/bridge"
+)
+
+// TODO remove this after the relay.go full refactoring
+
+type bridgeStub struct {
+	pendingBatchCallIndex          int
+	pendingBatches                 []*bridge.Batch
+	wasProposedTransfer            bool
+	lastProposedBatch              *bridge.Batch
+	lastWasProposedTransferBatchId bridge.BatchId
+	lastSignedActionId             bridge.ActionId
+	signersCount                   uint
+	lastExecutedActionId           bridge.ActionId
+	wasExecuted                    bool
+	proposeTransferActionId        bridge.ActionId
+	proposeTransferError           error
+	proposedStatusBatch            *bridge.Batch
+	proposeSetStatusActionId       bridge.ActionId
+
+	proposeSetStatusMutex sync.Mutex
+	proposeTransferMutex  sync.Mutex
+	signMutex             sync.Mutex
+	executeMutex          sync.Mutex
+}
+
+func (b *bridgeStub) lock() {
+	b.proposeSetStatusMutex.Lock()
+	b.proposeTransferMutex.Lock()
+	b.signMutex.Lock()
+	b.executeMutex.Lock()
+}
+
+// GetPending -
+func (b *bridgeStub) GetPending(_ context.Context) *bridge.Batch {
+	defer func() { b.pendingBatchCallIndex++ }()
```

Comment:
add function body on new line?

Reply:
also, should `pendingBatchCallIndex` be protected?

Reply:
Yes, that was the old code

Reply:
Fixed the stub